registry.json

```diff
@@ -2344,6 +2344,38 @@
       "clientDocs": "https://eth.wiki/json-rpc/API"
     }
   },
+  {
+    "id": "secret",
+    "name": "Secret",
+    "displayName": "Secret",
+    "coinId": 529,
+    "symbol": "SCRT",
+    "decimals": 6,
+    "blockchain": "Cosmos",
+    "derivation": [
+      {
+        "path": "m/44'/529'/0'/0/0"
```

Comment:
I'm not sure why the derivation path is different than cosmos, when I'm using keplr + ledger, I'm validating Secret transaction using the cosmos app, could you please clarify?

Reply:
@Milerius Secret Network is using a different derivation path (like [Kava](https://github.com/trustwallet/wallet-core/blob/master/registry.json#L1184) and [Crypto.org](https://github.com/trustwallet/wallet-core/blob/master/registry.json#L2264) for that matter)
When you use Keplr (hot wallet), it uses the right path.
If you want to use ledger with the right derivation path, you would need to use the Secret Network Ledger app.

Reply:
> @Milerius Secret Network is using a different derivation path (like [Kava](https://github.com/trustwallet/wallet-core/blob/master/registry.json#L1184) and [Crypto.org](https://github.com/trustwallet/wallet-core/blob/master/registry.json#L2264) for that matter) When you use Keplr (hot wallet), it uses the right path. If you want to use ledger with the right derivation path, you would need to use the Secret Network Ledger app.

Thanks for the explanation